### PR TITLE
APP-3018: Using varargs in constructor and setAttachment methods

### PR DESCRIPTION
**Pull request by symphony-hong**

### CODE REVIEW

**Review by symphony-elias — APPROVED** [review]
LGTM

File: symphony-bdk-legacy/symphony-api-client-java/src/main/java/model/OutboundMessage.java [hunk 1]
```diff
@@ -23,37 +24,25 @@ public OutboundMessage(String message, String data) {
         this.data = data;
     }
 
-    public OutboundMessage(String message, File[] attachment) {
+    public OutboundMessage(String message, File... attachment) {
```
Comment (symphony-thibault) [inline symphony-bdk-legacy/symphony-api-client-java/src/main/java/model/OutboundMessage.java:27]: What happen if we pass a `null` value here?

Comment (symphony-hong) [reply symphony-bdk-legacy/symphony-api-client-java/src/main/java/model/OutboundMessage.java:27]: We have to cast it to File class, then the attachment array will contain only 1 null element.

**Review by ystan- — APPROVED** [review]
LGTM

**Pull request merged**